packages/node_modules/@webex/internal-plugin-dss/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@webex/internal-plugin-dss",
+  "version": "2.16.0",
+  "description": "",
+  "license": "MIT",
+  "author": "Colin Read <coread@cisco.com>",
+  "main": "dist/index.js",
+  "devMain": "src/index.ts",
+  "repository": "https://github.com/webex/webex-js-sdk/tree/master/packages/node_modules/@webex/internal-plugin-dss",
+  "engines": {
+    "node": ">=8"
+  },
+  "browserify": {
+    "transform": [
+      "babelify",
```

Comment:
we are planning to add yarn so it would be better if we start adding the sub dependency in here , for this plugin core and mercury is the only dependency

Reply:
done